test/commands/package/packageVersion.nut.ts

```diff
@@ -345,7 +346,11 @@ describe('package:version:*', () => {
       ];
       expect(output).to.have.length.greaterThan(0);
       expect(output?.at(0)).to.have.keys(keys);
+      // @ts-ignore
+      const codeCoverage = output?.[0]?.CodeCoverage?.toString() || '';
+      expect(codeCoverage).to.equal('use --verbose for code coverage');
```

Comment:
```suggestion
      expect(output[0].CodeCoverage).to.equal...
```

this should already be a `string` (no .toString() required)
what's the `ts-ignore` ignoring?

Reply:
ts-ignore i was using locally to test something, removed both.